src/scss/styles.css

```diff
@@ -0,0 +1,36 @@
+html {
```

Comment:
Should these styles be in your base.scss file?

Reply:
I did try adding these to the base.scss and changing the stylesheet link to base.scss, but the styles did not load when I viewed the page.

I think I need to research how scss works to better understand why I need to move the styles to base.scss, and what else I am missing re: they are not displaying when I move them. I am going to commit the other changes for you though. I'm working Saturday so I'll spend some time figuring out how the base.scss works. Thank you! 😊